masl/parser/src/MaslWalker.g

```diff
@@ -1396,7 +1396,8 @@ domainServiceDefinition//[DomainService service]
                                    returnType?
                                    codeBlock
                                                             {
-                                                                args[0] = getFile();
+                                                                //args[0] = getFile();
```

Comment:
In this file, why are we carrying the commented out lines forwards.  I think they should be deleted.